articles/api-auth/tutorials/adoption/delegation.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Delegation and the OIDC-conformant pipeline
+---
+
+<%= include('./_about.md') %>
+
+# Delegation and the OIDC-conformant pipeline
+
+[Delegation](/api/authentication#delegation) is used for many operations, depending on your particular use case:
+
+* Exchanging an ID token issued to one client for a new one issued to a different client
```

Comment:
I mentioned this below to explain why delegation was deprecated:

> Given that ID tokens should no longer be used as API tokens and that refresh tokens should be used only at the token endpoint, this endpoint is now considered deprecated.

I think it's more useful to reinforce the "ID tokens are not access tokens" concept than go into detail about legacy usage of representing APIs as clients and such, especially since delegation is a relatively advanced/complex use case. What do you think @twistedstream ?